src/main/java/duke/logic/commands/UserSetup.java

```diff
@@ -71,7 +71,7 @@ public void initialise(String info) {
             }
             setGender(sex);
         } else if (user.getActivityLevel() == 5) {
-            int activity = 0;
+            int activity = 5;
```

Comment:
Why was this changed?

Reply:
activity level ranges from 0 to 4 in the program while it's displayed at 1-5 in UI output. therefore putting an impossible value can help to check for validation as int cannot be null